### ISSUE TITLE
[Asura Scans] Connector not working

### ISSUE THREAD

**Issue by esih351h** [manga-download/hakuneko#2302]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

When Refresh Manga List Popup Message
Failed to receive content from "https://asurascans.com/wp-admin/admin-ajax.php" (status: 400) - 

1. Asura Scans
4. Error: Failed to receive content from "https://asurascans.com/wp-admin/admin-ajax.php" (status: 400) - 
    at Connector.mjs:453
    at async AsuraScans._getMangasFromPage (WordPressMadara.mjs:46)
    at async AsuraScans._getMangas (WordPressMadara.mjs:38)
    at async AsuraScans._getMangaList (Connector.mjs:557) AsuraScans {id: "asurascans", label: "Asura Scans", tags: Array(2), isLocked: false, initialized: true, …}

![Screenshot (3)](https://user-images.githubusercontent.com/6265698/93282997-ce671180-f7f9-11ea-95be-716c79ddf1c1.png)

 - Version 6.17

**Comment by 27136773** [comment]
Website changed Template
